markdown/events/2022/bytesize-gitpod.md

```diff
@@ -25,3 +25,121 @@ It is our hope that these talks / videos will build an archive of training mater
+
+[12:06](https://www.youtube.com/watch?v=kBoC6QBU-M0t=726)
+The other thing, now it's finished. In our results file we have our results and if we look in MultiQC, for example, we have the MultiQC report. Again, you can use these preview buttons, so that you can render these particular results into a nice file where you can see everything that's going on. Here is the results of that FASTQC result in MultiQC. I think this is quite cool to know that you can you can just quickly run these pipelines.
+
+[12:36](https://www.youtube.com/watch?v=kBoC6QBU-M0t=756)
+Within this particular environment you might want to change something. So maybe you're particularly interested in changing a particular module. You can still change everything in a Gitpod environment and once you change something... I'll just add... I won't actually push this to the branch, because people won't be happy. But if I add on `cat` or whatever I've changed here. If I'm changing the code here you'll notice on the left hand side, on the source control, it's noticed that one of the files within your repository has changed. If we click this button it said that we've changed the main.nf and you can now use the git functions within Gitpod to add that particular change and you can also then make a message for that particular change. Also, down the bottom you can see that we have this development branch - on the bottom left-hand side. If we click this we can also choose to create a new branch. If you've made changes to the code this is a very quick way to create a new branch and show that we can push it back to the repository and then do a pull request.
+
+[13:47](https://www.youtube.com/watch?v=kBoC6QBU-M0t=827)
+I think that's everything I really wanted to show you on this very basic example. I'm going to quickly go to another example that I think is quite helpful to explore, which is actually on the website. If I go to the website repository we can also run it from here. I'll just show a few more features that I think are really interesting to see. We can actually live render the whole website and show that the whole website is changing and we can change the underlying markdown. This is quite helpful when we've been developing the website so hopefully this will quickly load and I can show you this example.
```

Comment:
```suggestion
I think that's everything I really wanted to show you in this very basic example. I'm going to quickly go to another example that I think is quite helpful to explore, which is actually on the website. If I go to the website repository we can also run it from here. I'll just show a few more features that I think are really interesting to see. We can actually live render the whole website and show that the whole website is changing and we can change the underlying markdown. This is quite helpful when we've been developing the website so hopefully, this will quickly load and I can show you this example.
```